sceptre/strides/config/prod/github-oidc-sage-bionetworks-it-admin.yaml

```diff
@@ -1,14 +1,16 @@
 template:
   type: http
-  url: https://raw.githubusercontent.com/Sage-Bionetworks/aws-infra/v0.4.1/templates/IAM/github-oidc-provider.j2
-stack_name: github-oidc-sage-bionetworks-it
+  url: https://raw.githubusercontent.com/Sage-Bionetworks/aws-infra/v0.7.6/templates/IAM/github-oidc-provider.j2
+stack_name: github-oidc-sage-bionetworks-it-admin
```

Comment:
what's the purpose to renaming this stack?  The stack name for the equivalent OIDC for our organization accounts is called `github-oidc-sage-bionetworks-it` take a look at the usage in [GH action file](https://github.com/Sage-Bionetworks-IT/organizations-infra/blob/master/.github/workflows/main.yaml)